### PR TITLE
Add deploy methods

### PR DESCRIPTION
**Pull request by jdorweiler**
Adds methods to create and update a deploy status. 
https://developer.github.com/v3/repos/deployments

### CODE REVIEW

**Comment by fayland** [comment]
Hi

could you help add POD as well?

Thanks

File: lib/Net/GitHub/V3/Repos.pm [hunk 1]
```diff
@@ -197,6 +197,10 @@ my %__methods = (
     # http://developer.github.com/v3/repos/statuses/
     list_statuses => { url => "/repos/%s/%s/statuses/%s" },
     create_status => { url => "/repos/%s/%s/statuses/%s", method => 'POST', args => 1 },
+
+    # https://developer.github.com/v3/repos/deployments
+    create_deploy => { url => "/repos/%s/%s/deployments/", method => 'POST', args => 1 },
+    update_deploy => { url => "/repos/%s/%s/deployments/%s/statuses", method => 'POST', args => 1 },
```
Comment (fayland) [inline lib/Net/GitHub/V3/Repos.pm]: that's not very correct.

there should be 4 methods. list_deployments, create_deployment, list_deployment_statuses, create_deplayment_status

two GET two POST

**Comment by jdorweiler** [comment]
@fayland I added POD and added the other methods.

**Comment by fayland** [comment]
Thanks for the patch. and it's released.

**Comment by jdorweiler** [comment]
Thanks! :cake:

**Pull request merged**